Crash on Save

RustRover 2024.3.7

```
java.lang.NoClassDefFoundError: Could not initialize class com.dioxuslabs.dioxus.native.DioxusIntellij$RustLibrary
	at com.dioxuslabs.dioxus.native.DioxusIntellij.format(DioxusIntellij.kt:17)
	at com.dioxuslabs.dioxus.actions.DioxusCheckOnSaveAction$processDocuments$1$1.run(DioxusCheckOnSaveAction.kt:48)
	at com.intellij.openapi.progress.impl.CoreProgressManager.startTask(CoreProgressManager.java:491)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.startTask(ProgressManagerImpl.java:133)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcessWithProgressAsynchronously$7(CoreProgressManager.java:542)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$4(ProgressRunner.java:249)
	at com.intellij.openapi.progress.ProgressManager.lambda$runProcess$0(ProgressManager.java:98)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$1(CoreProgressManager.java:223)
	at com.intellij.platform.diagnostic.telemetry.helpers.TraceKt.use(trace.kt:45)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:222)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$14(CoreProgressManager.java:674)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:749)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:705)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:673)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:79)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:203)
	at com.intellij.openapi.progress.ProgressManager.runProcess(ProgressManager.java:98)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$5(ProgressRunner.java:249)
	at com.intellij.openapi.progress.impl.ProgressRunner$ProgressRunnable.run(ProgressRunner.java:502)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$launchTask$18(ProgressRunner.java:467)
	at com.intellij.util.concurrency.ChildContext$runInChildContext$1.invoke(propagation.kt:103)
	at com.intellij.util.concurrency.ChildContext$runInChildContext$1.invoke(propagation.kt:103)
	at com.intellij.util.concurrency.ChildContext.runInChildContext(propagation.kt:109)
	at com.intellij.util.concurrency.ChildContext.runInChildContext(propagation.kt:103)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$launchTask$19(ProgressRunner.java:463)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:735)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:732)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:732)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.lang.ExceptionInInitializerError: Exception java.lang.UnsatisfiedLinkError: Unable to load library 'dioxus':
libdioxus.so: cannot open shared object file: No such file or directory
libdioxus.so: cannot open shared object file: No such file or directory
Native library (linux-x86-64/libdioxus.so) not found in resource path (/opt/rustrover/lib/platform-loader.jar:/opt/rustrover/lib/util-8.jar:/opt/rustrover/lib/util.jar:/opt/rustrover/lib/app-client.jar:/opt/rustrover/lib/util_rt.jar:/opt/rustrover/lib/product.jar:/opt/rustrover/lib/opentelemetry.jar:/opt/rustrover/lib/app.jar:/opt/rustrover/lib/product-client.jar:/opt/rustrover/lib/lib-client.jar:/opt/rustrover/lib/stats.jar:/opt/rustrover/lib/jps-model.jar:/opt/rustrover/lib/external-system-rt.jar:/opt/rustrover/lib/rd.jar:/opt/rustrover/lib/bouncy-castle.jar:/opt/rustrover/lib/protobuf.jar:/opt/rustrover/lib/forms_rt.jar:/opt/rustrover/lib/lib.jar:/opt/rustrover/lib/externalProcess-rt.jar:/opt/rustrover/lib/groovy.jar:/opt/rustrover/lib/annotations.jar:/opt/rustrover/lib/jsch-agent.jar:/opt/rustrover/lib/kotlinx-coroutines-slf4j-1.8.0-intellij.jar:/opt/rustrover/lib/nio-fs.jar:/opt/rustrover/lib/trove.jar) [in thread "ApplicationImpl pooled thread 5"]
	at com.sun.jna.NativeLibrary.loadLibrary(NativeLibrary.java:325)
	at com.sun.jna.NativeLibrary.getInstance(NativeLibrary.java:481)
	at com.sun.jna.Library$Handler.<init>(Library.java:197)
	at com.sun.jna.Native.load(Native.java:618)
	at com.sun.jna.Native.load(Native.java:592)
	at com.dioxuslabs.dioxus.native.DioxusIntellij$RustLibrary$Companion.<clinit>(DioxusIntellij.kt:10)
	at com.dioxuslabs.dioxus.native.DioxusIntellij$RustLibrary.<clinit>(DioxusIntellij.kt)
	... 32 more
```

--- a/README.md
+++ b/README.md
@@ -1,7 +1,8 @@
-<!-- Plugin description -->
 # Dioxus JetBrains IDEs Plugin
 
 ![Dioxus Splash](https://github.com/DioxusLabs/dioxus/raw/main/notes/dioxus_splash_8.avif)
+
+<!-- Plugin description -->
 
 This plugin wraps functionality in Dioxus CLI to be used in your JetBrains IDEs!
 
@@ -15,4 +16,5 @@ This overview provides a brief introduction to Dioxus. For a more in-depth guide
 
 - [Getting Started](https://dioxuslabs.com/learn/0.5/getting_started)
 - [Book (0.5)](https://dioxuslabs.com/learn/0.5)
+
 <!-- Plugin description end -->

